scripts/gen-rook-kustomize.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -ex
+
+# expect that the common.sh is in the same dir as the calling script
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+. ${SCRIPTPATH}/common.sh --source-only
+
+if [ -z "$CEPH_IMAGE" ]; then
```

Comment:
Is this actually `IMAGE` instead here?  Seems like it might be based on [1].

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/8d26cd388b9423cf2cb79d854499b5e0ef80397f/Makefile#L2169

Reply:
oh right, thank you!

Reply:
Done